js/package.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "google-protobuf",
+  "version": "3.0.0-alpha.1",
```

Comment:
We use "3.0.0-alpha.5" for this release. Alpha-1 to alpha-4 are already taken by previous releases (beta-1 == alpha-4) and we'd better not reuse them (otherwise JS alpha-1 will not work with protoc alpha-1 and it will be very confusing).